### ISSUE TITLE
model's parameters going to be empty

### ISSUE THREAD

**Issue by G-Bong** [mrlooi/maskrcnn-benchmark#19]
## ❓ Questions and Help

Hello all

When I do execution the code below in my ubuntu terminal, I get the error **RuntimeError: stack expects a non-empty TensorList**. I am trying to train the model using MS-COCO 2017 datasets (train2017, val2017, test2017).
`python tools/train_net.py --config-file "configs/rotated/e2e_ms_rcnn_R_50_FPN_1x.yaml" SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025 SOLVER.MAX_ITER 720000 SOLVER.STEPS "(480000, 640000)" TEST.IMS_PER_BATCH 1`

![image](https://user-images.githubusercontent.com/21220998/74218577-bec89d00-4ced-11ea-87a7-4175f89df3b0.png)

This is my dependencies for this **rotated_maskrcnn**.
![image](https://user-images.githubusercontent.com/21220998/74218678-f6cfe000-4ced-11ea-80da-9304fd18d9c9.png)

Could you give me any tips?

Best regards

Thanks.

**Comment by mrlooi** [comment]
From the traceback it would seem like your model.parameters() is empty.
Either that or clip_grad_norm has changed API. This repo has been tested on 1.0, 1.1. Not tested on torch 1.5

**Comment by G-Bong** [comment]
@mrlooi I am curious that what version of python did you use? I am working with conda to follow your step-by-step installation but it is not working in my docker of nvidia/cuda-9.0-devel

**Comment by mrlooi** [comment]
It should work with python 2 and 3. Have you changed your torch version to 1.0 or 1.1?

**Comment by G-Bong** [comment]
@mrlooi Yes I tried to run your code after changing the torch version. I tried many times but I am still stuck for the environment setting. Now I am trying to use your Docker files.
I failed to make docker image from your Dockerfile in dockerfile folder. Now I am trying with docker-jupyter.

Could you give me comment consistently when I give you errors??
I am very worried that you are getting irritated by my questions.

Anyway, thank you so much for your comment so far. :)

**Comment by mrlooi** [comment]
Hey no problem, I wouldn't use the docker file, it's not updated.
If I were you, I'd have a look into the model.parameters() output, and see if it's empty. If it's empty, it means perhaps your model didn't load properly somewhere. If not, it's probably still the API change and perhaps your torch version is still not correct? Double check that your interpreter is using the right torch version.  
At the end, if all else fails, try running on python2.7 or python3.5/3.6. Sorry I can't be of much more help here
